scripts/main.js

```diff
@@ -67,4 +31,8 @@ projectPopUp.exitBtn.addEventListener(('click'), () => {
   projectPopUp.closePopUp();
 });
 
-window.addEventListener('load', init());
+window.addEventListener('load', initProjects());
+
+formRef.addEventListener('keyup', () => {
```

Comment:
- [ ] **Please make sure that the validation is triggered when the user clicks the submit button.**

Screenshot from the project requirements
:------------------------------------------------------:
![image](https://user-images.githubusercontent.com/59168430/123340239-8a175780-d569-11eb-9cd9-815ef0cd11c4.png)